### ISSUE TITLE
Custom sounds don't play after a while of being added

### ISSUE THREAD

**Issue by Konglomneshued** [rafaelmardojai/blanket#306]
When I add custom sounds they play fine for a couple days and then they stop functioning and I have to delete and re-add them.

I'm on Fedora 38 and installed the Flatpak.  The audio clips I'm adding are mp3 and are several hours long.

**Comment by rafaelmardojai** [comment]
Yeah, this is a known issue I have seen since some time ago. It is probably related to the documents portal used on Flatpak, where the files becomes not longer available to the app after some time. 

I guess you could workaround this giving Blanket permission to you home folder using [Flatseal](https://flathub.org/apps/com.github.tchx84.Flatseal) or the Flatpak CLI.

I recently have added a check to not load files that not longer exist in the portal fuse file system [aaa9b19edc29a6e708608b06404174bcec4aecfd], but I need to investigate this further to report it upstream.
